dit/data_utils.py

```diff
@@ -43,7 +46,7 @@ def time_spent_on(logbook):
 # integer
 if os.path.isfile('DIT_TESTING'):
 
-    BASE_NOW = datetime(2016, 9, 10, 18, 57, 49, 0)
+    BASE_NOW = datetime(2016, 9, 10, 18, 57, 49, 0, tzinfo=pytz.utc)
```

Comment:
For instance: `tzinfo=timezone(timedelta(-1, 79200), 'BRST')`, which would be consistent with our current `datetime.now(get_localzone())`.